src/chime_dash/assets/app.css

```diff
@@ -71,3 +71,7 @@ input[type="number"]::-webkit-outer-spin-button {
 	margin-top: calc(-.75rem - 1px); /* 1px connects with pointer */
 }
 /* end date picker wrangling */
+
+button#copy_params, button#sidebar_params {
```

Comment:
I recommend generalizing into a class for buttons in the sidebar (e.g., `button.sidebar-secondary-action`) rather than tying style specifically to the element ids